### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by hlandau**
Rough draft of a new roadmap document so we at least have something to communicate for now.

We need to decide what to put in the FIPS section. Taking suggestions here.

(If @anton wants to take over this PR, that's fine also)

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -3,70 +3,36 @@ breadcrumb: Roadmap
...
-1. Argon2 KDFs ([RFC 9106]; openssl/openssl#12255 & openssl/openssl#12256)
-2. Attribute Certificates ([RFC 5755]; openssl/openssl#15857)
-3. Hybrid Public Key Encryption ([RFC 9180]; openssl/openssl#17172)
-4. Raw Public Keys ([RFC 7250]; openssl/openssl#16620)
-
-[RFC 5755]: https://datatracker.ietf.org/doc/html/rfc5755
-[RFC 7250]: https://datatracker.ietf.org/doc/html/rfc7250
-[RFC 9106]: https://datatracker.ietf.org/doc/html/rfc9106
-[RFC 9180]: https://datatracker.ietf.org/doc/html/rfc9180
+TODO TALK ABOUT FIPS
```
Comment (t8m) [inline roadmap.md]: I would remove the section for now. Or just mention that we are still working on FIPS 140-3 validation of OpenSSL. I would not mention a particular version.

**Review by t8m — CHANGES_REQUESTED** [review]

**Comment by hlandau** [comment]
How's this?

**Review by t8m — APPROVED** [review]

**Comment by arapov** [comment]
@hlandau, it is good; thanks for capturing it; it is a big help for me.

Please keep this PR open until the end of the week at the latest. I'll see if I want to add something to the text.

**Comment by DDvO** [comment]
I just wonder why the section "**Post-quantum cryptography**" has been removed.
Doesn't this include helpful clarification and a nice hint to use the Open Quantum Safe provider?

File: roadmap.md [hunk 2]
```diff
@@ -3,70 +3,36 @@ breadcrumb: Roadmap
 ---
 # OpenSSL project roadmap
```
Comment (JefriReynaldi) [inline roadmap.md:4]:
```dart

```

**Review by JefriReynaldi — APPROVED** [review]

**Comment by kroeckx** [comment]
This doesn't really look like a roadmap. I hope we plan to do more than just quic, maybe performance, and fips.

**Comment by t8m** [comment]
> This doesn't really look like a roadmap. I hope we plan to do more than just quic, maybe performance, and fips.

It is very brief but for 3.3 we do not have that much time left and for 3.4 we really need the release steering committee to decide. I think this should be merged and then updated soon after 3.4 plans are established.

**Comment by t8m** [comment]
@arapov please approve

**Comment by arapov** [comment]
> This doesn't really look like a roadmap. I hope we plan to do more than just quic, maybe performance, and fips.

We don't commit to anything but QUIC, although performance and fips are top priorities that are worked on.

**Comment by arapov** [comment]
> @arapov please approve

Not yet. I need to update it.

**Comment by kroeckx** [comment]
Why should this only cover 3.3? There are lots of things we want to do. We don't know when, but we do plan to do it at some point.

**Comment by ghost** [comment]
@arapov Anton is about to update this issue.

**Comment by ghost** [comment]
Clean up the page. @arapov

**Comment by arapov** [comment]
closed in favour of #446

**Pull request closed without merging**